modules/indexer/issues/indexer.go

```diff
@@ -19,6 +19,7 @@ import (
 	"code.gitea.io/gitea/modules/queue"
 	"code.gitea.io/gitea/modules/setting"
 	"code.gitea.io/gitea/modules/util"
+	"github.com/blevesearch/bleve/v2"
```

Comment:
Please add an empty line ahead of this one

Reply:
Our own `codeformat` should do the work with `make fmt`, but now our own `codefomtat` isn't called. We should invite it back 😊

Reply:
on it .-..